static/app/components/events/profileEventEvidence.tsx

```diff
@@ -12,32 +14,58 @@ export const ProfileEventEvidence = ({event, projectSlug}: ProfileEvidenceProps)
   const evidenceData = event.occurrence?.evidenceData ?? {};
   const evidenceDisplay = event.occurrence?.evidenceDisplay ?? [];
 
-  const frameName = evidenceData.frameName;
-  const framePackage = evidenceData.framePackage;
-
   const keyValueListData = [
-    {
-      subject: 'Profile ID',
-      key: 'Profile ID',
-      value: (
-        <pre>
-          <Link
-            to={generateProfileFlamechartRouteWithHighlightFrame({
-              profileId: event.id,
-              projectSlug,
-              orgSlug: organization.slug,
-              frameName,
-              framePackage,
-              query: {
-                referrer: 'issue-details',
-              },
-            })}
-          >
-            {event.id}
-          </Link>
-        </pre>
-      ),
-    },
+    ...(evidenceData.transactionId && evidenceData.transactionName
+      ? [
+          {
+            subject: 'Transaction Name',
+            key: 'Transaction Name',
+            value: (
+              <pre>
+                <Link
+                  to={getTransactionDetailsUrl(
+                    organization.slug,
+                    generateEventSlug({
+                      id: evidenceData.transactionId,
+                      project: projectSlug,
+                    }),
+                    undefined,
+                    {referrer: 'issue'}
```

Comment:
where is `referrer` used? is it passed into the backend with any requests? not sure about this case, but in the backend there's a referrers enum since we don't allow any arbitrary referrers. Just flagging this because I see referrer changed from `issue-details` to `issue`

Reply:
`referrer` here is just being used for front end analytics so someone could see how much traffic is being sent to profiles from issues (theoretically - afaik no one needs this yet, but I was anticipating that we might at some point). I thought `issue` seemed a bit better since this link can appear outside of issue details.